models/messages.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var Messages = sequelize.define('Messages', {
+    content: DataTypes.TEXT,
+    senderId: DataTypes.INTEGER,
```

Comment:
Actually normally you should have these in the associate block as associations rather than declaring them here